reviews-block.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Registers all block assets so that they can be enqueued through the block editor
+ * in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/block-editor/tutorials/block-tutorial/applying-styles-with-stylesheets/
+ */
+class WP_Review_Block {
+	public function register() {
+		$this->register_block();
+	}
+
+	function register_block() {
+		$dir = dirname( __FILE__ );
+
+		$script_asset_path = "$dir/build/index.asset.php";
+		if ( ! file_exists( $script_asset_path ) ) {
+			throw new Error(
+				'You need to run `npm start` or `npm run build` for the "create-block/review-block" block first.'
+			);
+		}
+		$index_js     = 'build/index.js';
+		$script_asset = require( $script_asset_path );
+		wp_register_script(
+			'create-block-review-block-block-editor',
```

Comment:
Use the name `wp-plugin-review-block-editor`. Do a review and remove reference to `create-block` everywhere.

Reply:
Fixed.